library/std/canon.qs

```diff
@@ -287,9 +284,7 @@ namespace Microsoft.Quantum.Canon {
     /// X(q);
     /// ```
     operation ApplyP(pauli : Pauli, target : Qubit) : Unit is Adj + Ctl {
-        if   pauli == PauliX { X(target); }
```

Comment:
Is there a way to override formatting for a block if one feels strongly about a particular formatting in a particular place? Something like #pragma(DoNotAutoformat) <- not an actual directive, just made it up.

Reply:
You can turn the "save on format" setting off specifically for Q#. Then you would only manually either format the whole file, in which case you could adjust the formatting after words, or you could format only the sections of the code you want via highlighting. Finally, if you are not wanting to use the formatter at all, there is a setting to disable the Q# formatter.

Reply:
There's currently no built-in way to format multiple files at once like that. You'd have to go to each individual file anyway to format each of them.

Reply:
This seems to me like an ask for an additional formatting feature, unless you disagree and believe such a change is integral to the other changes in this PR. We could consider adding this in the future if the demand for it is strong enough. Would you add a GitHub issue for this please?